README.md

```diff
@@ -1 +1,93 @@
-# Disney-Characters
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Fruitycee
+
+A web application for a provides data for all kinds of fruit.
+
+# Live Demo
+- Add Allow CORS Extension for the api to work.
+- Please click on Logo in the site to make it work.
+
+[Live Demo](https://muskan2532150.github.io/Fruitycee)
+
```

Comment:
Your project is supposed to be deployed using Netlify or Heroku not github pages. Please deploy your project using netlify or Heroku. Just incase you are unsure on how to deploy a react project using netlify kindly follow this link [here](https://www.freecodecamp.org/news/how-to-deploy-a-react-application-to-netlify-363b8a98a985/)

Reply:
@KayLemba, In the requirements there is nowhere mention that we can't deploy using Github pages

 [Project Requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md).

Reply:
I'd advise you to make the required changes we code reviewers have guidelines we follow I cannot ask you to make a change outside project requirements. 

Please Have a look.

![20221008_094407.jpg](https://user-images.githubusercontent.com/79506105/194696471-6f9302d9-62a3-4cdf-a635-e298a6a84f13.jpg)

Reply:
@KayLemba, I saw the Live version and on line 7 it says we can deploy using Heroku ,Netlily or GitHub Pages

Reply:
Ok then sorry about let me delete the change